include/bootstrap.php

```diff
@@ -3,6 +3,14 @@
 // Used for performance calculations
 $dStartTime = microtime(true);
 
+// set locale
+$language_code = 'ru_RU.utf8';
```

Comment:
Hi! Ok, I will move language settings to config and make English as default language. I thought about browser locale detection, but don't have time to implement it now.